app/assets/javascripts/landing.js

```diff
@@ -1,17 +1,44 @@
 $(function(){
   $('.up').hide()
+  $('.down').hide()
+
+  $('.button').on("mouseenter", function(){
+    $(".loopify").hide()
+    $("#loopify").show()
```

Comment:
These CSS things are confusing - you are hiding an element with a class `loopify` and doing the opposite with an element with the id `loopify`.

In all likelihood, this can be solved with CSS instead of javascript (which should always be preferred). Let me get to you css and make a suggestion there.